redfish-core/lib/systems.hpp

```diff
@@ -1625,6 +1626,12 @@ inline void
         return;
     }
     asyncResp->res.jsonValue["PowerMode"] = modeValue;
+
+    BMCWEB_LOG_DEBUG("Safe mode: {}", safeMode);
+    nlohmann::json& oemSafeMode = asyncResp->res.jsonValue["Oem"];
+    oemSafeMode["@odata.type"] = "#OemComputerSystem.Oem";
+    oemSafeMode["IBM"]["@odata.type"] = "#OemComputerSystem.IBM";
+    oemSafeMode["IBM"]["SafeMode"] = safeMode;
```

Comment:
If `safeMode` is defined as optional, this would be inside the check of it.
Also, the type would be better to be specific as in 1060 branch.

```
if(safeMode)
{
   BMCWEB_LOG_DEBUG("Safe mode: {}", *safeMode);
    nlohmann::json& oemSafeMode = asyncResp->res.jsonValue["Oem"];
    oemSafeMode["@odata.type"] = "#OemComputerSystem.Oem";
    oemSafeMode["IBM"]["@odata.type"] = "#OemComputerSystem.v1_0_0.IBM";   <--
    oemSafeMode["IBM"]["SafeMode"] = *safeMode;
}
```